server/controllers/bookController.js

```diff
@@ -134,6 +136,63 @@ class BookController {
       Utils.errorStatus(res, 500, error.message);
     }
   }
+
+  static async recieveBook(req, res){
+    const { isbn, patronId } = req.body;
+    const { damaged } = req.params;
```

Comment:
How do you get to know the condition of the said book before this function runs?. Because since the book is returned physically, the person handling it would definitely know the books condition, and submit a form to the server, i feel the damaged property should be on the request body,  instead of the params. But still explain to me how u came about this implementation

Reply:
What I thought is while returning the book you check the damaged property if it's damaged, 
So weather it's in the request body or in the param I really don't see a difference, both will still get to the controller.

Reply:
True, but this physical book will already have a damaged property set in the request params when returning it?

Reply:
If u look through the rest of the code you will see that the property is set to either true or false, depending on the scenario

Reply:
Jude just like using `req.params` sha. I agree with Dominic, it would have been better to have the damaged property also on the request body. Nonetheless, it still gets the job done.

Reply:
OK, it's 2 against 1. I'll do that once theres power